web-platform/website/src/components/pages/home/EmailNotVerifiedToast.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+
+import { Strings } from "../../../resources/Strings";
+import { useAuthentication } from "../../../services/AuthenticationService";
+import { ProductCheckEmailDialog } from "./ProductCheckEmailDialog";
+import { ToastComponent } from "./study-card/ToastComponent";
+
+const emailStrings = Strings.components.pages.home.alerts.verifyEmail;
+
+export function EmailNotVerifiedToast() {
+  const [showEmailDialog, setShowEmailDialog] = useState<boolean>(false);
+  const [showEmailNotVerifiedToast, setShowEmailNotVerifiedToast] =
+    useState<boolean>(false);
+  const { isUserVerified, reloadUser, sendEmailVerification } =
+    useAuthentication();
+
+  useEffect(() => {
+    reloadUser();
```

Comment:
`reloadUser()` is only needed in this `EmailNotVerifiedToast` since all it does is update the status of user verification. It can be removed from everywhere else (i.e. from the other two toasts and the ProductToast container itself)